ynput/ayon-unreal
# Replace ast.literal_eval with json.loads for creator and publish attributes to avoid crashes on Unreal

## Changelog Description
Ported from https://github.com/ynput/OpenPype/pull/6318


## Additional info
n/a


## Testing notes:

1. Ayon -> Publish
2. Create Instance
3. Turn on/off Instance, or turn on/off setting
4. Save Instance
5. Should be worked as expected
